.github/workflows/main.yml

```diff
@@ -0,0 +1,24 @@
+name: CI
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    name: Node.js ${{ matrix.node_version }} on ${{ matrix.os }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node_version:
+          - 12
+          - 10
+          - 8
```

Comment:
How about dropping Node 8 support?

Reply:
@papb We'll drop Node 8 support in a separate PR.